pmm-app/src/pmm-qan/panel/components/Details/database-models/mysql/service.ts

```diff
@@ -1,23 +1,7 @@
-import { apiRequestManagement } from 'shared/components/helpers/api';
+import { apiRequestActions } from 'shared/components/helpers/api';
 
 export default {
-  getShowCreateTableMySQL(body) {
```

Comment:
One option is to leave the service functions and adjust the payload in the service function bodies to match what the BE expects now.

Reply:
Updated.